tests/unit/Codeception/Module/Doctrine2Test.php

```diff
@@ -568,6 +565,10 @@ public function testOneToManyRecursiveEntityCreation()
 
     public function testHaveFakeRepository()
     {
+        if (version_compare(InstalledVersions::getVersion('doctrine/orm'), '3', '>=')) {
```

Comment:
[`EntityManager::$repositoryFactory`](https://github.com/doctrine/orm/blob/5b8b5f28f535e1f03b54dcfb0427407ed92f5b72/src/EntityManager.php#L87) was made `readonly` in ORM v3.